libbeat/common/transport/tlscommon/ca_pinning.go

```diff
@@ -48,15 +37,15 @@ type verifyPeerCertFunc func([][]byte, [][]*x509.Certificate) error
 // NOTE: Defining a PIN to check certificates is not a replacement for the normal TLS validations it's
 // an additional validation. In fact if you set `InsecureSkipVerify` to true and a PIN, the
 // verifiedChains variable will be empty and the added validation will fail.
-func MakeCAPinCallback(hashes pins) func([][]byte, [][]*x509.Certificate) error {
+func MakeCAPinCallback(hashes []string) func([][]byte, [][]*x509.Certificate) error {
```

Comment:
would it make sense to use a `common.StringSet` here?

Reply:
Not sure its really needed, hashes slice should be small.